tests/test_callbacks.py

```diff
@@ -30,11 +30,13 @@
 
 
 class HalfPairwiseJudge(BasePairwiseJudge):
-    """Naive pairwise judge that always returns [1, 0]"""
+    """Naive pairwise judge that always returns [1, 0] for two prompts"""
 
-    def judge(self, prompts, completions, shuffle_order=True):
+    def judge(self, prompts, completions, shuffle_order=True, return_scores=False):
         # just check that the batch size is 2
         assert len(prompts) == 2
+        if return_scores:
+            return [0.3, 0.7]
```

Comment:
```suggestion
            return [0.3, 0.9]
```

so that the average win prop is not equal to the win rate